test/integration/router/router_http_server.go

```diff
@@ -257,14 +264,13 @@ func (s *TestHttpService) startServing(addr string, handler http.Handler) error
 
 	s.listeners = append(s.listeners, listener)
 
-	fmt.Printf("Started, serving at %s\n", listener.Addr().String())
+	glog.Infof("Started, serving at %s\n", listener.Addr().String())
 
 	go func() {
 		err := http.Serve(listener, handler)
 
 		if err != nil {
-			fmt.Printf("Server message: %v", err)
-			s.Stop()
```

Comment:
Yes

On Sun, Jul 10, 2016 at 10:48 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In test/integration/router/router_http_server.go
> <https://github.com/openshift/origin/pull/9578#discussion_r70196464>:
>
> >
> >  	go func() {
> >  		err := http.Serve(listener, handler)
> >
> >  		if err != nil {
> > -			fmt.Printf("Server message: %v", err)
> > -			s.Stop()
>
> was stopping wrong before?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/12aa57c2488769a24af8d39431d5b8b81ff5464f..a850e5e780643ff932482d6a9224cbfbb137f332#r70196464>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_pyh5_Zq_TtanFhMR-F73xb87_gSIks5qUa7mgaJpZM4I-xTE>
> .
>